### ISSUE TITLE
can not write multip-layer to same the same geopackage

### ISSUE THREAD

**Issue by 861** [evetion/GeoDataFrames.jl#32]
if add a new layer to the existing geopackage, it will replace the current geopackage

**Comment by evetion** [comment]
Agreed, there's no such functionality now. Would an `append!` make more sense to you? Or a `write(... append=True)`? Or even, if a file already exists and the layername is different, append by default?

**Comment by 861** [comment]
default would be perfect！thanks！

**Comment by kavir1698** [comment]
Hello, is there any update on this issue?

**Comment by evetion** [comment]
Hi there, not right now, but given the interest, I might tackle it. Are you interested in appending to an existing table, or writing a new layer in a geopackage?

**Comment by kavir1698** [comment]
That would be great. I'd like to write multiple layers to a file.

**Comment by evetion** [comment]
v0.3.13 should be out in 20 minutes or so.

You can then do
```julia
GDF.write("test.gpkg", df; layer_name = "foo", update = true)
```

Which should add it as a new layer to the existing geopackage. Note you need to set the layer_name to something new, or pass a dict with "OVERWRITE"=>"YES" to the options keyword argument to overwrite the layer already present. There's no way to append to an existing layer currently.

**Comment by kavir1698** [comment]
Fantastic! Thanks for the quick implementation.
